fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckState.kt

```diff
@@ -31,8 +33,10 @@ sealed interface ReviewQualityCheckState : State {
     /**
      * Supported product retailers.
      */
-    enum class ProductVendor {
-        AMAZON, BEST_BUY, WALMART,
+    enum class ProductVendor(@StringRes val stringId: Int) {
+        AMAZON(stringId = R.string.review_quality_check_retailer_name_amazon),
```

Comment:
Let's not add R imports in State, we can do this mapping on the UI layer similar to Onboarding.